x/mint/keeper/keeper_test.go

```diff
@@ -340,7 +340,7 @@ func (suite *KeeperTestSuite) TestCreateDeveloperVestingModuleAccount() {
 
 			if tc.expectedError != nil {
 				suite.Error(actualError)
-				suite.Equal(actualError, tc.expectedError)
+				suite.ErrorIs(actualError, tc.expectedError)
```

Comment:
Is this used anywhere else in our codebase?

Also, I believe `suite.ErrorIs` just makes sure that the expected error is a _subset_ of the actual error (see testify implementation [here](https://github.com/stretchr/testify/blob/181cea6eab8b2de7071383eca4be32a424db38dd/assert/assertion_forward.go#L249)), whereas `suite.Equal` makes sure both are exactly equal, so I have a light preference for keeping this as `suite.Equal`.

Reply:
You want to use `errors.Is` (https://pkg.go.dev/errors#Is), so you can safely catch and compare the error, even if it's wrapped.

Reply:
Had a chat with @p0mvn about this and decided to go with `suite.ErrorIs` because it was very difficult to compare sdkerrors using `suite.Equal`. For instance this is showing unequal;
```
Not equal: 
expected: vesting module account doesnot exist: module account does not exist 
actual  : vesting module account doesnot exist: module account does not exist
                                    
    Diff:
    --- Expected
    +++ Actual
    @@ -8,7 +8,8 @@
       }),
    -  stack: (*errors.stack)((len=6) {
    +  stack: (*errors.stack)((len=7) {
        (uintptr) 0x4625d7b,
    -   (uintptr) 0x5061f88,
    -   (uintptr) 0x552c858,
    -   (uintptr) 0x5068896,
    +   (uintptr) 0x552c436,
    +   (uintptr) 0x40d1865,
    +   (uintptr) 0x40d0ddf,
    +   (uintptr) 0x5069656,
        (uintptr) 0x411a322,
```

Reply:
`suite.Equal()` would compare error addresses which are not equal, `suite.ErrorIs()` is desirable here